src/utils/index.js

```diff
@@ -32,34 +31,32 @@ export const playAudio = ({ audioFile, bgColor, waveColor, src, setSrc, setWebmV
   const mediaRecorder = new MediaRecorder(mediaStream, { mimeType })
   mediaRecorder.addEventListener('dataavailable', (e) => chunks.push(e.data))
 
-  fileReader.onload = () => {
-    audioContext.decodeAudioData(fileReader.result, (buffer) => {
-      if (src) {
-        src.stop()
-        cancelAnimationFrame(animationId)
-      }
+  const blob = URL.createObjectURL(audioFile)
+  const audioElement = new Audio(blob)
+  const source = audioContext.createMediaElementSource(audioElement)
 
-      const tempSrc = audioContext.createBufferSource()
-      setSrc(tempSrc)
+  audioElement.playbackRate = 1;
 
-      tempSrc.buffer = buffer
-      tempSrc.connect(analyser)
-      tempSrc.connect(destination)
+  audioElement.onloadstart = () => {
+    source.connect(analyser)
+    source.connect(destination)
 
-      mediaRecorder.start(100)
-      tempSrc.start(0)
-
-      animationId = requestAnimationFrame(render)
```

Comment:
I think that requestAnimationFrame is not needed (Because of 60fps delay ...). Therefore, animation starts by invoking `render`.